xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1177 @@
+   <important>
+    <para>
+     To make use of the helm charts, you must complete
+     <xref linkend="sec.deploy.scenarios.airgap.container-registry"/>.
+    </para>
+   </important>
+   <para>
+    The helm charts will require images available from a registry mirror. The
+    charts themselves are served on a simple webserver and do not require any
+    particular configuration apart from basic networking availability and a
+    hostname.
+   </para>
+   <sect3>
+    <title>Mirror Configuration</title>
+    <procedure>
+     <title>Setting Up Chart Repository Mirror</title>
+     <step>
+      <para>
+       We will re-use the webserver that is shipped with &rmtool; to create
+       a virtual host for the chart repository. The repository will be available
+       at <literal>http://mymirror.local:80/charts</literal>.
+      </para>
+     </step>
+    </procedure>
+    <para>
+     Update the Helm chart repository by following the shared update procedure
+     <xref linkend="sec.deploy.scenarios.airgap.update"/>.
+    </para>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.helm-charts.client">
+    <title>Client Configuration</title>
+    <para>
+     Add the webserver as a repo to <command>helm</command>.
+     </para>
+     <para>
+      This step needs to be performed on a machine where Helm is installed and
+      configured to talk to the Tiller server in the &productname; cluster.
+      Refer to, <xref linkend="sec.admin.software.helm"/>.
+     </para>
+     <para>
+      <literal>SUSE-MIRROR</literal> will be the name for this repository listed
+      by Helm and can be freely defined.
+     </para>
+<screen>&prompt.user;<command>helm repo add SUSE-MIRROR <replaceable>&lt;mymirror.local&gt;/charts</replaceable></command>
+   </screen>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.update">
+   <title>Updating Registry Mirror And Helm Charts</title>
+   <note>
+    <title>Live Update Of Registry</title>
+    <para>
+     There is no need to stop the container image registry services while doing
+     the update procedures. All changed images will be re-indexed automatically.
+    </para>
+   </note>
+   <para>
+    Helm charts and container images must be refreshed in the same procedure,
+    otherwise charts might refer to image versions that are not mirrored or you
+    are mirroring outdated image versions that cause the chart deployment to
+    fail.
+   </para>
+
+   <procedure>
+    <title>Pull Data From Upstream Sources</title>
+    <step>
+     <para>
+      SSH into the mirror server on the external network.
+     </para>
+    </step>
+    <step>
+     <para>
+     Download all charts from the repository to the filesystem
+     (e.g. <filename>/tmp/charts</filename>).
+     </para>
+     <para>
+      This action will download all charts and overwrite the existing Helm
+      chart repository URL. Replace <literal>https://mymirror.local/charts</literal>
```

Comment:
**warning:** right now they doc above states the charts are only served over HTTP. If that's the case then this command has to be changed from `https://` to `http://`.